editor/asset_library_editor_plugin.cpp

```diff
@@ -808,6 +808,7 @@ void EditorAssetLibrary::_request_image(ObjectID p_for, String p_image_url, Imag
 	iq.image_index = p_image_index;
 	iq.image_type = p_type;
 	iq.request = memnew(HTTPRequest);
+	id.asset_id = 0;
```

Comment:
Well, that's embarrassing. I will admit I stopped working on this and submitted the PR to gauge interest. I realize now I failed to compile and run the unit tests after coming to this decision :(